doc/api/readline.md

```diff
@@ -90,7 +90,7 @@ The `'pause'` event is emitted when one of the following occur:
 
 * The `input` stream is paused.
 * The `input` stream is not paused and receives the `SIGCONT` event. (See
-  events `SIGTSTP` and `SIGCONT`)
+  events [SIGTSTP][] and [SIGCONT][])
```

Comment:
@italoacasas Maybe I’m misunderstanding you, but what I mean is that they should be links *and* code blocks, i.e. ``[`SIGTSTP`][] and [`SIGCONT`][]`` (and at the bottom of the file ``[`SIGTSTP`]: readline.html#readline_event_sigtstp``, too). I’m totally agreeing that adding links makes sense here!